packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -1,34 +1,78 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+
 import type { AlertSlots, AlertState } from './Alert.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const alertClassName = 'fui-Alert';
 export const alertClassNames: SlotClassNames<AlertSlots> = {
   root: 'fui-Alert',
-  // TODO: add class names for all slots on AlertSlots.
-  // Should be of the form `<slotName>: 'fui-Alert__<slotName>`
+  icon: 'fui-Alert__icon',
+  action: 'fui-Alert__action',
 };
+export const intentClassName = 'fui-Alert__intent';
```

Comment:
I don't have any immediate recommendation, but I do not like this separate export.
As an engineer, I would expect to have all the classes accessible through the `alertClassNames` object, but this one is separate. (I know `intent` is not a slot, but anyway).

Reply:
Makes sense. @khmakoto / @layershifter, any suggestions?

Reply:
I replied in another thread, I don't think that we really need this `className` (it's used only in tests), so suggestion is simply remove it.

Reply:
Yeah, I agree with @layershifter, we should simply remove this.

Reply:
Removed!